### PR TITLE
Andymck/poc grpc v2 plus vals as chall

### PR DESCRIPTION
**Pull request by andymck**
Core side of https://github.com/helium/miner/pull/1469

### CODE REVIEW

File: include/blockchain_vars.hrl [hunk 1]
```diff
@@ -152,6 +152,13 @@
 %%       and so this value must be sufficient as to give time for absorb to occur
 -define(poc_receipts_absorb_timeout, poc_receipts_absorb_timeout).
 
+%% Number of POC ephemeral keys to be generated and included in a validator heartbeat: pos_integer
+-define(poc_validator_ephemeral_key_count, poc_validator_ephemeral_key_count).
```
Comment (Vagabond) [inline include/blockchain_vars.hrl:156]: is it possible to calculate this from the # of active validators and the "per block poc rate" instead?

Comment (Vagabond) [reply include/blockchain_vars.hrl:156]: I'm worried about having to manually change this value if the # of active validators changes

Comment (andymck) [reply include/blockchain_vars.hrl:156]: yeah that makes for a better approach

Comment (andymck) [reply include/blockchain_vars.hrl]: fixed here: 
https://github.com/helium/blockchain-core/pull/1272/commits/67480e4307e3e1bfaca1fb8a64a847639ce0d9ea
https://github.com/helium/miner/pull/1469/commits/cf381c6c4b2878d7d6747660ea49dc3036605853
https://github.com/helium/sibyl/pull/23/commits/f1a49b2908efa4aa217165e0aebd6d31ab7ad9b2

**Pull request merged**